schema/site.schema.json

```
@@ -959,6 +959,11 @@
       "minimum": 5,
       "!go": { "pointer": true }
     },
+    "permissions.syncJobCleanupInterval": {
+      "description": "Time interval (in seconds) of how often cleanup worker should remove old jobs from permissions sync jobs table.",
+      "type": "integer",
+      "default": 60
```

```suggestion
      "default": 60,
      "minimum": 1
```
To defend from cool hackers with their negative values

---

its taken care of in the code, but added here as well.